### PR TITLE
Zib-wounds.WoundCharacteristics

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**
TODO:

* https://bits.nictiz.nl/browse/ZIB-2100

### CODE REVIEW

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="39126-8" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:woundLength.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="WoundLength" />
+      <definition value="The length of the wound." />
```
Comment (LuudSlagter) [inline resources/zib/zib-wounds.WoundCharacteristics.xml:64]: Combine the definitions present in Wound and PressureUlcer, i.e. 'The length of the (decubitus) wound.'. This also applies to the definitions of WoundDepth and WoundWidth.

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    <element id="Observation.component:woundTissue">
+      <path value="Observation.component" />
+      <sliceName value="woundTissue" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:woundTissue.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
```
Comment (LuudSlagter) [inline resources/zib/zib-wounds.WoundCharacteristics.xml:155]: Maybe add `.code` _waiting on ZIB-2100_ so we don't forget to change this later on (not necessary of course).

File: release-notes.md [hunk 3]
```diff
@@ -385,4 +385,8 @@ This document contains release notes per zib, indicating differences with their
 
 ## zib-VisualFunction
 * Changed fixed slice on `.code.coding` to a pattern on `.code`.
-* Relaxed cardinality of `value[x]` to 0..1 of the conceptual cardinalities of the zib.
\ No newline at end of file
+* Relaxed cardinality of `value[x]` to 0..1 of the conceptual cardinalities of the zib.
+
+## zib-wounds.WoundCharacteristics
+* The Profile's name has been changed from "WoundCharacteristics" to "zib-wounds.WoundCharacteristics".
```
Comment (LuudSlagter) [inline release-notes.md:391]: profile's

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 4]
```diff
@@ -0,0 +1,270 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.WoundCharacteristics" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-wounds.WoundCharacteristics" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundCharacteristics.xml:4]: We select to use the word wounds without a capital letter.

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This WoundObservable profile caputures mutliple concepts such as wound length, width and depth which are used in two different HCIMs:&#xD;&#xA;&#xD;&#xA;* [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))&#xD;&#xA;* [PressureUlcer v3.4(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN))" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundCharacteristics.xml:17]: Fix

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+  <purpose value="A seperate Observation is deemed necessary for the observable properties of wounds." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundCharacteristics.xml:21]: Fix

File: release-notes.md (hunk 3 above)
Comment (Ashwin-nictiz) [reply release-notes.md:391]: Fix

File: resources/zib/zib-wounds.WoundCharacteristics.xml [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This WoundObservable profile caputures mutliple concepts such as wound length, width and depth which are used in two different HCIMs:&#xD;&#xA;&#xD;&#xA;* [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))&#xD;&#xA;* [PressureUlcer v3.4(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN))" />
+  <purpose value="A seperate Observation is deemed necessary for the observable properties of wounds." />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-wounds.WoundCharacteristics.xml:18]: Fix

**Pull request closed without merging**